intranet/static/js/dashboard/events.js

```diff
@@ -12,32 +12,40 @@ $(function() {
 
         var hidden = event.hasClass("hidden");
         var action = hidden ? "show" : "hide";
-        $.post("/events/" + action + "?" + id, {event_id: id}, function() {
-            console.info("event "+id+" "+action);
+
+        $.post("/events/" + action + "?" + id, {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["$" is not defined.](https://www.codacy.com/app/tjhsst-ion/ion/file/2755452134/issues/source?bid=3212528&fileBranchId=3355681#l16)